src/App.tsx

```diff
@@ -142,7 +170,16 @@ function App({ isEmbedded = false }: { isEmbedded?: boolean }) {
                 >
                   {!(embed?.isEmbedded && embed.mode === EmbedMode.Panels) && (
                     <CanvasProvider value={canvasService}>
-                      <CanvasView />
+                      <CanvasView
+                        shouldEnableZoomOutButton={shouldEnableZoomOutButton}
+                        shouldEnableZoomInButton={shouldEnableZoomInButton}
+                        canShowWelcomeMessage={canShowWelcomeMessage}
+                        showControls={showControls}
+                        showZoomButtonsInEmbed={showZoomButtonsInEmbed}
+                        showPanButtonInEmbed={showPanButtonInEmbed}
+                        isEmbedded={embed?.isEmbedded}
```

Comment:
q: wasn't `CanvasView` able to just read this from the context like before? it is provided here, after all 🤔

Reply:
My guy, stop reviewing this! It's in draft

Reply:
<img src="https://media.giphy.com/media/PBQGYblzEyGF5cIsJE/giphy.gif" />